packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -76,6 +76,10 @@ import PrimaryEmailVerified from '../../pages/Signup/PrimaryEmailVerified';
 import SignupConfirmed from '../../pages/Signup/SignupConfirmed';
 import WebChannelExample from '../../pages/WebChannelExample';
 import LinkValidator from '../LinkValidator';
+import ResetPasswordStartContainer from '../../pages/ResetPasswordRedesign/ResetPasswordStart/container';
+import ConfirmResetPasswordOtpContainer from '../../pages/ResetPasswordRedesign/ConfirmResetPasswordOtp/container';
```

Comment:
Since these are already in a new directory `ResetPasswordRedesign` that I assume we'll get rid of after we're stable, should we follow our convention of the directory name and component names roughly matching our route names, and then just rename the import here if needed? Having `Otp` especially in the names feels funky to me.